pkg/sensors/handler.go

```diff
@@ -181,6 +182,46 @@ func (h *handler) listTracingPolicies(op *tracingPolicyList) error {
 	return nil
 }
 
+func (h *handler) disableTracingPolicy(op *tracingPolicyDisable) error {
+	col, exists := h.collections[op.name]
+	if !exists {
+		return fmt.Errorf("tracing policy %s does not exist", op.name)
+	}
+
+	if !col.enabled {
+		return fmt.Errorf("tracing policy %s is already disabled", op.name)
+	}
+
+	err := col.unload()
+	if err != nil {
+		col.err = fmt.Errorf("failed to unload tracing policy: %w", err)
+		return err
+	}
+
+	col.enabled = false
+	h.collections[op.name] = col
```

Comment:
ah... that's tricky.
Maybe it would have been better to use a pointer in the map.

Reply:
indeed, but I think it's okay!